app/vmalert/datasource/client_vlogs.go

```diff
@@ -0,0 +1,58 @@
+package datasource
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+func (s *Client) setVLogsInstantReqParams(r *http.Request, query string, timestamp time.Time) {
+	// there is no type path prefix in victorialogs APIs right now, ignore appendTypePrefix.
+	if !*disablePathAppend {
+		r.URL.Path += "/select/logsql/stats_query"
+	}
+	q := r.URL.Query()
+	q.Set("time", timestamp.Format(time.RFC3339))
+	if s.applyIntervalAsTimeFilter && s.evaluationInterval > 0 {
```

Comment:
Would you mind mentioning the reason for doing this here and, probably, here https://docs.victoriametrics.com/victorialogs/querying/#querying-log-stats, because vlogs docs don't mention start/end params.